### PR TITLE
Fix color in displayShow when manually searching.

### PR DESCRIPTION
**Pull request by abeloin**
Changed function getManualSearchStatus to include overview.
Changed all attr to prop

### CODE REVIEW

File: sickbeard/webserve.py [hunk 1]
```diff
@@ -1809,6 +1809,7 @@ def searchEpisode(self, show=None, season=None, episode=None):
     def getManualSearchStatus(self, show=None, season=None):
         def getEpisodes(searchThread, searchstatus):
             results = []
+            showObj = sickbeard.helpers.findCertainShow(sickbeard.showList, int(show))
```
Comment (fernandog) [inline sickbeard/webserve.py]: is this related? 
https://github.com/SiCKRAGETV/sickrage-issues/issues/684

**Comment by fernandog** [comment]
@abeloin sorry reverting this. Couldnt fix it.

**Comment by abeloin** [comment]
@fernandog I'll redo the patch. It's mainly broken because the epAjaxSearch.js sends the word undefined instead of show integer when the values doesn't exists.
